Resources/Prototypes/Nyanotrasen/Entities/Clothing/Masks/masks.yml

```diff
@@ -28,3 +28,18 @@
     sprite: Nyanotrasen/Clothing/Mask/mask_oni_red.rsi
   - type: Clothing
     sprite: Nyanotrasen/Clothing/Mask/mask_oni_red.rsi
+
+- type: entity
+  parent: ClothingMaskPullableBase
+  id: FelinidCone
```

Comment:
If this descends from a clothing prototype, I'd prefer it follows the convention of other children. I.e. there was a commit pulled in the recent cherrypick that made `HatBandBlack` etc. into `ClothingHeadBandBlack`.

In this case, you'd make it `ClothingMaskFelinidCone`.

That said, this probably makes sense in a neck slot, but if it's in a mask slot, it'd block wearing gas masks too which seems to make sense given the bulk of the item?

I'm not sure on this one. If it's a mask, it shouldn't be pullable, at least. Those things probably aren't easy to remove.